cache/redis.go

```diff
@@ -49,11 +49,34 @@ func NewRedis(uri string, opts ...RedisOptionsFunc) (Cache, error) {
 		return nil, err
 	}
 
+	uo := &redis.UniversalOptions{
+		Addrs:         []string{o.Addr},
+		RouteRandomly: true,
+	}
+
+	for _, opt := range opts {
+		opt(uo)
+	}
+
+	c := redis.NewUniversalClient(uo)
+
+	return &redisCache{
+		client:  c,
+		decoder: stringDecoder{},
+	}, nil
+}
+
+func NewRedisUniversal(uri []string, opts ...RedisOptionsFunc) (Cache, error) {
```

Comment:
rename `uri` to `addrs` to be clear what you are passing in here.

Reply:
OK